tests/functional/object/mcg/test_mcg_replication_target_unreachable.py

```
@@ -0,0 +1,135 @@
+        4. Verify alert properties and bucket names (not hash IDs - DFBUGS-6380)
+        5. Delete the source bucket
+        6. Verify alert clears after source bucket deletion (blocked by DFBUGS-6398)
+        """
+        # 1. Create source and target buckets with a replication policy
+        target_bucket = bucket_factory(1, "OC")[0]
+        replication_policy = ("basic-replication-rule", target_bucket.name, None)
+        source_bucket = bucket_factory(1, "OC", replication_policy=replication_policy)[
+            0
+        ]
+        logger.info(
+            f"Replication configured: {source_bucket.name} -> {target_bucket.name}"
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)